Project.toml

```
@@ -3,21 +3,19 @@ uuid = "6f1fad26-d15e-5dc8-ae53-837a1d7b8c9f"
 license = "MIT"
 desc = "Tape based task copying in Turing"
 repo = "https://github.com/TuringLang/Libtask.jl.git"
-version = "0.7.0"
-
+version = "0.8.0"
```

Let's change this version later so the integration tests can run. 
```suggestion
version = "0.7.0"
```